sdk/communication/Azure.Communication.CallAutomation/src/Models/CallInvite.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.CallAutomation
+{
+    /// <summary>
+    /// Call invitee details.
+    /// </summary>
+    public class CallInvite
+    {
+        /// <summary>
+        /// Creates a new CallInvite object.
+        /// </summary>
+        /// <param name="targetPhoneNumberIdentity"></param>
+        /// <param name="callerIdNumber"></param>
+        public CallInvite(PhoneNumberIdentifier targetPhoneNumberIdentity, PhoneNumberIdentifier callerIdNumber)
+        {
+            Target = targetPhoneNumberIdentity;
+            SourceCallerIdNumber = callerIdNumber;
+        }
+
+        /// <summary>
+        /// Creates a new CallInvite object.
+        /// </summary>
+        /// <param name="targetIdentity"></param>
+        public CallInvite(CommunicationUserIdentifier targetIdentity)
+        {
+            Target = targetIdentity;
+        }
+
+        /// <summary>
+        /// Creates a new CallInvite object.
+        /// </summary>
+        /// <param name="targetIdentity"></param>
+        public CallInvite(MicrosoftTeamsUserIdentifier targetIdentity)
```

Comment:
nit: how come we use targetPhoneNumberIdentity as the name for PhoneNumberIdentifier but targetIdentity for both CommunicationUserIdentifier and MicrosoftTeamsUserIdentifier?

Reply:
we can change the name. Do you prefer to have targetIdentity for all or something like targetTeamsIdentity

Reply:
hmmm, i think for consistency, we can name them all to targetIdentity?